pages/k8s/equinix.md

```diff
@@ -76,7 +76,7 @@ machines:
     constraints: mem=32G
 applications:
   calico:
-    charm: cs:~containers/calico-826
+    charm: calico
```

Comment:
I thought I had already changed these, but I see that I haven't. Maybe there is such a thing as too much gin